### PR TITLE
[BUG FIX] Fix CircleCI for Fetching and Spectral Connectivity Saving After averaging frequencies

### PR DESCRIPTION
**Pull request by adam2392**
PR Description
--------------

Closes: #87
Closes: #92 

In collaboration with [weiglszonja](https://github.com/weiglszonja).

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-connectivity/blob/main/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

**Comment by adam2392** [comment]
@[weiglszonja](https://github.com/mne-tools/mne-connectivity/issues?q=is%3Aissue+is%3Aopen+author%3Aweiglszonja) FYI you can copy the changes in the `test_spectral.py::test_spectral_connectivity` and `epochs.py` files and verify it works for your unit test / use-case.

Then if you open your PR, we can work on merging it into the codebase?

File: doc/authors.inc [hunk 1]
```diff
@@ -3,4 +3,5 @@
 .. _Britta Westner: https://github.com/britta-wstnr
 .. _Alexander Kroner: https://github.com/alexanderkroner
 .. _Richard Höchenberger: https://github.com/hoechenberger
-.. _Alex Rockhill: https://github.com/alexrockhill
\ No newline at end of file
+.. _Alex Rockhill: https://github.com/alexrockhill
+.. _Szonja Weigl: https://github.com/weiglszonja
```
Comment (adam2392) [inline doc/authors.inc:7]: @weiglszonja lmk if this is correct?

File: doc/whats_new.rst [hunk 2]
```diff
@@ -29,7 +29,7 @@ Enhancements
 Bug
 ~~~
 
--
+- Fix the output of :func:`mne_connectivity.spectral_connectivity_epochs` when ``faverage=True``, allowing one to save the Connectivity object, by `Adam Li`_ and `Szonja Weigl`_ :gh:`91`
```
Comment (adam2392) [inline doc/whats_new.rst:32]: Same here: @weiglszonja lmk if this is correct?

File: doc/authors.inc (hunk 1 above)
Comment (weiglszonja) [reply doc/authors.inc:7]: It is, thanks!

**Pull request merged**